src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-caching.mdx

```diff
@@ -138,9 +164,10 @@ The ability to customize both the browser cache TTL and edge cache TTL allows yo
 
 ## Stale cache
 
-When your application's cache expires, it becomes *stale* (obsolete) and the server searches the origin for the most up-to-date version. This new version, with an updated expiration date, overwrites the old one (*stale*).
+**Stale cache** is an optimization to guarantee content delivery when a cache object is not used frequently. Traditionally, if the cached content expires or cannot be located, the edge requests content directly from origin servers.
+
+When stale cache is active and origin servers cannot be reached, a version of the content with expired TTL and the STALE status can be delivered for the request. The next successful request to the origin for the updated content then receives a HIT status.
```

Comment:
@arturdj Ajustada a descrição

Reply:
Not very accurate. Please re-check with Product Manager